admin/common_auth/admin.py

```diff
@@ -39,10 +39,6 @@ def send_email_invitation(self, request, queryset):
     send_email_invitation.short_description = 'Send email invitation to selected users'
 
     def save_model(self, request, obj, form, change):
```

Comment:
You can just remove this method. You're not doing anything the grandfather class method isn't doing.